### ISSUE TITLE
Doesn't build with Xcode 13 RC

### ISSUE THREAD

**Issue by shuguenot** [yonaskolb/XcodeGen#1127]
- clone project (master)
- run `swift run xcodegen`

```
'Spectre' /Users/xxx/XcodeGen/.build/checkouts/Spectre: warning: Invalid Exclude '/Users/xxx/XcodeGen/.build/checkouts/Spectre/Sources/Spectre/XCTest@4.swift': File not found.
/Users/xxx/XcodeGen/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Task.swift:286:37: error: value of optional type 'UnsafeMutablePointer<CChar>?' (aka 'Optional<UnsafeMutablePointer<Int8>>') must be unwrapped to a value of type 'UnsafeMutablePointer<CChar>' (aka 'UnsafeMutablePointer<Int8>')
        defer { argv.forEach { free($0)} }
                                    ^
/Users/xxx/XcodeGen/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Task.swift:286:37: note: coalesce using '??' to provide a default when the optional value contains 'nil'
        defer { argv.forEach { free($0)} }
                                    ^
                                       ?? <#default value#>
/Users/xxx/XcodeGen/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Task.swift:286:37: note: force-unwrap using '!' to abort execution if the optional value contains 'nil'
        defer { argv.forEach { free($0)} }
                                    ^
                                      !
/Users/xxx/XcodeGen/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Task.swift:300:55: error: value of optional type 'UnsafeMutablePointer<Int8>?' must be unwrapped to a value of type 'UnsafeMutablePointer<Int8>'
                    free(UnsafeMutableRawPointer(pair.pointee))
                                                      ^
/Users/xxx/XcodeGen/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Task.swift:300:55: note: coalesce using '??' to provide a default when the optional value contains 'nil'
                    free(UnsafeMutableRawPointer(pair.pointee))
                                                      ^
                                                              ?? <#default value#>
/Users/xxx/XcodeGen/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Task.swift:300:55: note: force-unwrap using '!' to abort execution if the optional value contains 'nil'
                    free(UnsafeMutableRawPointer(pair.pointee))
                                                      ^
                                                             !
/Users/xxx/XcodeGen/.build/checkouts/PathKit/Sources/PathKit.swift:583:12: error: value of optional type 'UnsafeMutablePointer<CChar>?' (aka 'Optional<UnsafeMutablePointer<Int8>>') must be unwrapped to a value of type 'UnsafeMutablePointer<CChar>' (aka 'UnsafeMutablePointer<Int8>')
      free(cPattern)
           ^
/Users/xxx/XcodeGen/.build/checkouts/PathKit/Sources/PathKit.swift:583:12: note: coalesce using '??' to provide a default when the optional value contains 'nil'
      free(cPattern)
           ^
                    ?? <#default value#>
/Users/xxx/XcodeGen/.build/checkouts/PathKit/Sources/PathKit.swift:583:12: note: force-unwrap using '!' to abort execution if the optional value contains 'nil'
      free(cPattern)
           ^
                   !
[35/63] Compiling Yams Encoder.swift
error: fatalError
```

**Comment by TheCoordinator** [comment]
@yonaskolb Seems the issues are mainly on SwiftCLI and PathKit packages. Let us know if we can help expedite this as it's breaking workflows and quite urgent to be fixed.

**Comment by yonaskolb** [comment]
Opening a PR on SwiftCLI would help. I can see there's one for PathKit already https://github.com/kylef/PathKit/pull/78

**Comment by TheCoordinator** [comment]
@yonaskolb Cool here we go https://github.com/jakeheis/SwiftCLI/pull/105

**Comment by TheCoordinator** [comment]
@yonaskolb SwiftCLI updated here https://github.com/jakeheis/SwiftCLI/releases/tag/6.0.3

**Comment by venturafonso** [comment]
Fix for pathKit merged [here](https://github.com/kylef/PathKit/pull/80)

**Comment by raptorxcz** [comment]
Waiting for https://github.com/tuist/XcodeProj/pull/643 update.

**Comment by gormster** [comment]
XcodeProj has been [fixed at 8.3.0](https://github.com/tuist/XcodeProj/releases/tag/8.3.0)

**Comment by 0111b** [comment]
@gormster can't see any problems in XcodeProj 8.3.0

**Comment by raptorxcz** [comment]
I prepared PR https://github.com/yonaskolb/XcodeGen/pull/1129.

**Comment by yonaskolb** [comment]
Fix released in 2.25.0. Thanks all!
